app/uploaders/spotlight/featured_image_uploader.rb

```diff
@@ -9,7 +9,7 @@ def extension_whitelist
     end
 
     def store_dir
-      "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+      "#{Spotlight::Engine.config.upload_dir}/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
```

Comment:
Quick question about this.  It was previously "uploads/...." and was already putting the records in the `public` directory.  Will changing this from "uploads/..." to "public/uploads/..." cause a redundant `public` directory inside the already existing `public` directory, or does CarrierWave figure that out for us?  Just trying to ensure that this wont cause any issues for upgrading existing spotlight apps.

Reply:
Previously `public` was hardcoded here, and I removed it when I made the change you're highlighting: https://github.com/projectblacklight/spotlight/pull/2096/files#diff-7d4f437731e1a8742d3acb9a9b78b3c7L54

I think that's the only place that uses `store_dir`, so changing it to explicitly include the `public` base directory should be OK, right?  Are there any other uses that I overlooked?

Reply:
Okay, I just spun up a test app using your fork of spotlight and uploaded an image.  That image ended up in `/Projects/spotlight/dunn-sl/.internal_test_app/public/public/uploads/spotlight/featured_image/image/1` (notice the `/public/public`.

This is what I'm trying to avoid here.  Previously it was just `uploads` (which put an uploads directory in `public`), and not it is `public/uploads` which puts it in a redundant `public` directory within `public`.